src/vendor/edlib/CTestTestfile.cmake

```diff
@@ -0,0 +1,8 @@
+# CMake generated Testfile for 
```

Comment:
We do not want to change anything in the the vendor subdirs. Can you plase not add files to stage via "add -A" (or the equivslent in the gui).  In gour case, you need to manually select which files to add. Please unstage any vendor libs
before committing or just dont stage them.